requirements/default.txt

```diff
@@ -1,6 +1,6 @@
-numpy>=1.17.3
-scipy>=1.4.1
-networkx>=2.2
+numpy>=1.19
+scipy>=1.8
+networkx>=2.7rc1
```

Comment:
@stefanv @jarrodmillman I'm a bit confused about this update. Do we really want to bump our dependencies to the bleeding edge SciPy and networkx straight away? I would have expected nx to implement some compatibility layer for a couple of versions at least...

Reply:
You could just do
```
 networkx>=2.2,<3.0
```
(not tested).

Reply:
But then when you release the next version of scikit-image won't work with the new version of NetworkX.

Reply:
I don't think that will fix your CI system.  You will need to udate it to not error on the deprecation warning.

Reply:
> I don't think that will fix your CI system. You will need to update it to not error on the deprecation warning.

Any specific warnings can be added to the ones being skipped in this function (there are examples for `tifffile`, `Pillow`, etc)
https://github.com/scikit-image/scikit-image/blob/main/skimage/_shared/testing.py#L198